### ISSUE TITLE
could not find upgrades contrabandcybernetics, nor debrisgambit

### ISSUE THREAD

**Issue by patof** [baranidlo/bench#151]
<img width="1262" alt="screen shot 2018-05-12 at 11 21 48" src="https://user-images.githubusercontent.com/10265795/39958872-1a17e59a-55d7-11e8-8943-e9f8dca4b099.png">

**Comment by Sleet01** [comment]
Yep, those are not yet implemented but are in common rotation on the tournament scene.

I'll try to get some cards implemented after this bug fix pass finishes up.
